lib/inferno/apps/cli/main.rb

```diff
@@ -99,6 +99,10 @@ def start
 
       desc 'suites', 'List available test suites'
       def suites
+        ENV['NO_DB'] = 'true'
+
+        require_relative '../../../inferno'
```

Comment:
This doesn't seem to make any difference in my machine; `bundle exec ./bin/inferno suites` works on main and this branch while `./bin/inferno suites` doesn't work on either branch. Same thing happens if I run each branch from a test kit.